src/style.css

```diff
@@ -0,0 +1,146 @@
+
+:root {
+  --color-dark: #060b35;
+  --color-light: #616070;
+  --color-blue: #00acdc;
+}
+
+h1,
+h2,
+h3,
+p,
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  max-width: 200px;
+  padding: 16px;
+  flex-direction: column;
+  background: #ffff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 0;
+}
+
+.card__product-name {
+  color: var(--color-dark);
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product-code {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-light);
+  font-style: normal;
+}
+
+.card__rate-container {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__reviews-number {
+  font-weight: 400;
```

Comment:
font-weight 400 by default, redundant

Reply:
* Not fixed, no need to add default font styles like font-weight: 400 and font-style: normal